client/src/components/ElementEditor/Header.js

```diff
@@ -40,6 +42,26 @@ class Header extends Component {
     }
   }
 
+  /**
+   * Handle the publication of a block, passing the block ID and version number in
+   */
+  handlePublish(event) {
+    event.stopPropagation();
+
+    const { id, version, actions: { handlePublishBlock } } = this.props;
+
+    const publishMessage = i18n._t(
+      'ElementHeader.CONFIRM_PUBLISH',
+      'Are you sure you want to publish this block?'
+    );
+
+    // eslint-disable-next-line no-alert
+    if (handlePublishBlock && confirm(publishMessage)) {
```

Comment:
I feel like we don't normally confirm publishes. I think we can drop the confirm here. /cc @clarkepaul

Reply:
Good point! Let's see what @clarkepaul says.

Reply:
@ScopeyNZ Thanks for spotting this. I checked with @clarkepaul and we do not need confirmation since we haven't confirmed publishing elements before. I will take this bit out.